subprojects/execution/src/main/java/org/gradle/internal/execution/steps/SkipUpToDateStep.java

```diff
@@ -143,7 +142,7 @@ public Try<Execution> getExecution() {
     private void logExecutionReasons(List<String> reasons, UnitOfWork work) {
         if (LOGGER.isInfoEnabled()) {
             Formatter formatter = new Formatter();
-            formatter.format("%s is not up-to-date because:", StringUtils.capitalize(work.getDisplayName()));
+            formatter.format("Executing %s because:", work.getDisplayName());
```

Comment:
That still may be not true: We may load the task from the cache, and then we won't be executing the task. So I am not sure this message would be better.

Reply:
How about `Not skipping execution of %s because:`?

Reply:
That's more in line with the message we give when we _do_ skip:

    Skipping %s as it is up-to-date.

Reply:
I don't now what skipping means. Would you also say we skip the task when we load it from the build cache? My main point here is that I don't feel like any of the proposed alternatives are better than the original. The problem is that we mark a task as UP-TO-DATE when it didn't do any work after executing the task action. I think that is causing the confusion. Should both of theses states be called up-to-date? And if yes, what are we checking in `SkipUpToDateStep`?

Reply:
Based on [docs](https://docs.gradle.org/current/userguide/more_about_tasks.html) SKIPPED is:
> Task did not execute its actions.

> * Task has been explicitly excluded from the command-line. See [Excluding tasks from execution](https://docs.gradle.org/current/userguide/command_line_interface.html#sec:excluding_tasks_from_the_command_line).

> * Task has an onlyIf predicate return false. See [Using a predicate](https://docs.gradle.org/current/userguide/more_about_tasks.html#sec:using_a_predicate).

so that could be a confusing verb too.